ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource

```diff
@@ -100,5 +100,6 @@ Select Runtime API Protocol
     [Documentation]    Selects which API protocol for the runtime
     [Arguments]    ${protocol}=gRPC
     Click Element    xpath://button[@data-testid="custom-serving-api-protocol-selection"]
-    Wait Until Page Contains Element    xpath://li[@data-testid="dropdown-item ${protocol}"]
-    Click Element    xpath://li[@data-testid="dropdown-item ${protocol}"]
+    Sleep    10s
```

Comment:
try not to use sleep where possible. You would add timeout in the next wait keyword

Reply:
okay

Reply:
replaced sleep with timeout